### ISSUE TITLE
New format: German Highlander

### ISSUE THREAD

**Issue by Akamayu** [magefree/mage#6302]
Hi there, would it be possible to add on the platform another format called German Highlander ? It's another variant of Highlander (minimum 100 cards decks, each card allowed only once except basic lands, no sideboard), simply using a banned card list instead of a point list (like Canadian and Australian versions do). 

An updated website gathers all information about it, especially the current banned cards list : http://www.highlandermagic.info/index.php?id=bannedlist

Implementing this new format on the platform would be great, so thanks in advance !

**Comment by Zerrisx** [comment]
Duplicate of #6766

**Comment by ddzn** [comment]
Now that I implemented #7335 we can close this issue.
